src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -282,7 +282,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_do_irecv(void *buf,
 
         /* Unpack */
         MPIDI_OFI_REQUEST(rreq, event_id) = MPIDI_OFI_EVENT_RECV_PACK;
-        MPIDI_OFI_REQUEST(rreq, noncontig.pack.pack_buffer) = MPL_malloc(data_sz, MPL_MEM_OTHER);
+        MPIDI_OFI_REQUEST(rreq, noncontig.pack.pack_buffer) =
+            MPL_aligned_alloc(256, data_sz, MPL_MEM_OTHER);
```

Comment:
I think the alignment parameter is byte. Are we sure we want to align to 256B?

Reply:
Ok, I will change alignment to 64.